packages/next/src/views/Version/RenderFieldsToDiff/FieldDiffCollapser/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { ChevronIcon } from '@payloadcms/ui'
+import React, { useState } from 'react'
+
+import Label from '../Label/index.js'
+import './index.scss'
+
+const baseClass = 'field-diff-collapser'
+
+type Props = {
+  children: React.ReactNode
+  initCollapsed?: boolean
+  label: React.ReactNode
+}
+
+const FieldDiffCollapser: React.FC<Props> = ({ children, initCollapsed = false, label }) => {
```

Comment:
Named this component "Collapser" to minimize confusion with the Collapsible field. This component can be used in multiple field components, including the Collapsible field component.